app/views/shared/_image_tags_and_actions.erb

```diff
@@ -1,13 +1,25 @@
-<div class="js-image-tags-and-actions row">
+<div class="js-image-tags-and-actions row bg-faded py-2">
   <span class="js-tags-list col-10 justify-content-start">
     <% image.tag_list.each do |tag| %>
       <%= link_to(tag, "#{images_path}?tag_name=#{tag}", :class=>"js-card-tag badge badge-default") %>
     <% end %>
   </span>
-  <span class="js-card-btn-group btn-group col-2 justify-content-end">
-    <%= button_to 'Delete', image_path(image),
+  <span class="js-card-btn-group btn-group col-2 justify-content-end" role="group">
+    <%= button_to new_image_image_share_path(image),
+                  method: :get,
+                  #data: { confirm: 'Are you sure you want to delete this image?' },
```

Comment:
shall we remove this line if it's not needed?

Reply:
Yes